app/src/main/java/com/example/polina/socialnetwork/SearchActivity.java

```diff
@@ -12,67 +14,79 @@
 import android.widget.AdapterView;
 import android.widget.BaseAdapter;
 import android.widget.EditText;
-import android.widget.ImageButton;
 import android.widget.ListView;
 import android.widget.TextView;
+import android.widget.Toast;
+
 import com.android.volley.toolbox.ImageLoader;
 import com.android.volley.toolbox.NetworkImageView;
+
 import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
+
 import java.util.ArrayList;
 
 
 /**
  * Created by polina on 22.09.15.
  */
-public class SearchFragment extends Fragment {
+
+public class SearchActivity extends Activity {
 
     SNApp snApp;
-    Context thisContext;
     ArrayList<User> users = new ArrayList<>();
     SearchAdapter adapter;
     EditText searchName;
+    private SharedPreferences sharedPreferences;
 
 
-
-    public View onCreateView(LayoutInflater inflater, ViewGroup container,
-                             Bundle savedInstanceState) {
-        View v = inflater.inflate(R.layout.fragment_search, null);
-        snApp= (SNApp) getActivity().getApplication();
-        searchName = (EditText) v.findViewById(R.id.search_name);
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        sharedPreferences = getSharedPreferences(Utils.PROFILE_PREFERENCES, MODE_PRIVATE);
```

Comment:
add getSupportActionBar().setDisplayHomeAsUpEnabled(true)  here. and don't forget to handle it in 
@Override
    public boolean onOptionsItemSelected(MenuItem item) {
       .......................................
        switch (item.getItemId()) {
            case android.R.id.home:
                onBackPressed();
        }
        return super.onOptionsItemSelected(item);